code/obj/machinery/computer/card.dm

```diff
@@ -448,9 +453,14 @@
 			var/obj/item/paper/P = new /obj/item/paper
 			P.set_loc(src.loc)
 
-			var/t1 = "<B>Crew Manifest:</B><BR>"
+			var/t1 = "<B>Crew Manifest:</B><hr>"
+			var/stored = ""
+			if(length(by_type[/obj/cryotron]))
+				var/obj/cryotron/cryo_unit = pick(by_type[/obj/cryotron])
+				for(var/L as anything in cryo_unit.stored_crew_names)
+					stored += "<i>- [L]<i><br>"
 			for(var/datum/db_record/t as anything in data_core.general.records)
-				t1 += "<B>[t["name"]]</B> - [t["rank"]]<BR>"
+				t1 += "<B>[t["name"]]</B> - [t["rank"]]<br><br><b>In Cryogenic Storage:</b><hr>[stored]<BR>"
```

Comment:
Uhh I'm pretty sure this one adds the full list of people in cryo after *every living person*, you probably only want to do that once.

Reply:
Hm, yeah this is a bit different from all the other manifest renderers. I'm gonna work on fixing this up; I must've fucked something up when replacing some of the code

Reply:
![image](https://user-images.githubusercontent.com/28785926/139513471-c0391134-ccad-42a5-93dd-b6db3a8058b0.png)
Seems to work fine now after fixing it up